sys/ztimer/core.c

```diff
@@ -106,6 +109,13 @@ static void _add_entry_to_list(ztimer_clock_t *clock, ztimer_base_t *entry)
 
     ztimer_base_t *list = &clock->list;
 
+#ifdef MODULE_PM_LAYERED
+    /* First timer on the clock's linked list */
+    if (list->next == NULL && clock->required_pm_mode > -1) {
```

Comment:
Just for clarification: `list->next == NULL` is required, to check if it is the first entry, correct?

Reply:
Nevermind, didn't see the `.next` below.